gemmapy/sdk/api/default_api.py

```diff
@@ -1535,7 +1535,7 @@ def get_datasets(self, **kwargs):  # noqa: E501
         >>> result = thread.get()
 
         :param bool async_req:
-        :param str query:
+        :param QueryArg query:
```

Comment:
@oganm This is the main highlight of this release. The query is still a string though.